src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write your code here
+const populationElements = document.querySelectorAll('.population');
+
+const populations = Array.from(populationElements).map((el) => {
+  const number = Number(el.textContent.replace(/,/g, ''));
+
+  return isNaN(number) ? 0 : number;
+});
+
+const totalPopulation = populations.reduce(
+  (sum, population) => sum + population,
+  0,
+);
+const averagePopulation = Math.round(totalPopulation / populations.length);
```

Comment:
There is a potential division by zero error here if there are no elements with the class 'population'. Consider adding a check to ensure that 'populations.length' is not zero before performing the division.